git_patrol.py

```diff
@@ -167,8 +165,8 @@ def log_command_error(command, returncode, stdout_bytes, stderr_bytes):
     git_ref: The git ref (ex: refs/heads/master, refs/tags/v0.0.1) that
       triggered this workflow execution.
   Returns:
-    The UUID of the newly created Cloud Build workflow if successful. Otherwise
-    returns None.
+    The in-progress Cloud Build workflow state as a JSON string if successful.
```

Comment:
Provide link to schema, or document here?

Reply:
Done.